src/cnd.h

```diff
@@ -0,0 +1,19 @@
+#ifndef RLANG_CND_H
+#define RLANG_CND_H
+
+#include <stdbool.h>
+
+
+void r_inform(const char* fmt, ...);
+void r_warn(const char* fmt, ...);
+void r_abort(const char* fmt, ...);
+
+SEXP interp_str(const char* fmt, ...);
```

Comment:
Why not just make it `r_`? But even `rlang_` is not that bad as you'll get autocomplete.  

Anyway, that's the way C APIs have to be, and there's no point in fighting convention.